### PR TITLE
Aiidalab app register

### PR DESCRIPTION
**Pull request by pablo-de-andres**

### CODE REVIEW

File: docs/source/apps/aiidalab.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
+# Resident AiiDAlab app
+
+## AiiDAlab registration
+
+In order to register an AiiDaLab app on the platform, you first need to have the app registered in the AiiDAlab registry.
+Please check the [AiiDAlab App developer guide](https://aiidalab.readthedocs.io/en/latest/app_development/index.html) to learn how to create your AiiDAlab app and publish it so it can be installed from [AiiDAlab's MarketPlace deployment](https://materials-marketplace.aiidalab.net/).
```
Comment (yoavnash) [inline docs/source/apps/aiidalab.md:6]: Can we publish the SimPARTIX aiidalab app in github as a reference?
Even better would be to make it generic and add it to the get-started app, but that's perhaps too much for now.

File: docs/source/apps/aiidalab.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    bearerAuth:
+      type: http
+      scheme: bearer
+      bearerFormat: JWT
+```
+````
+
+For the host app, select the previously registered AiiDAlab platform application.
+
+![AiiDAlab resident App registration](../_static/img/app_registration/aiidalab.png)
```
Comment (yoavnash) [inline docs/source/apps/aiidalab.md:52]: Should we also explain how to interact with the marketplace from within aiidalab once the app is deployed there? For example, how to create a frontend link to my voila/app-mode application.

File: docs/source/apps/aiidalab.md (hunk 1 above)
Comment (pablo-de-andres) [reply docs/source/apps/aiidalab.md:6]: I literally just replied to an email regarding this :)
The MICRESS app is almost completely updated, once that is done we will update the UC1 and publish it.

File: docs/source/apps/aiidalab.md (hunk 2 above)
Comment (pablo-de-andres) [reply docs/source/apps/aiidalab.md:52]: Not sure what you mean about the link, but in general the interaction is via the Python SDK. But that is relevant for the app development itself, not the registration.

File: docs/source/apps/aiidalab.md (hunk 1 above)
Comment (yoavnash) [reply docs/source/apps/aiidalab.md:6]: Should we wait awhile longer then? (until it's complete)

File: docs/source/apps/aiidalab.md (hunk 2 above)
Comment (yoavnash) [reply docs/source/apps/aiidalab.md:52]: I just remembered that aiidalab can't give you a link to an already deployed app that only shows the frontend and accessible to other users.

File: docs/source/apps/aiidalab.md (hunk 1 above)
Comment (pablo-de-andres) [reply docs/source/apps/aiidalab.md:6]: I would personally merge this so we don't have another blocked task waiting, but it is not a strong opinion. 
We might also want to add more documentation later, as to how to develop or register an app on AiiDAlab.

**Review by yoavnash — APPROVED** [review]

**Pull request merged**